lib/devcycle-ruby-server-sdk/api/devcycle_api.rb

```diff
@@ -64,7 +64,7 @@ def all_features(user_data, opts = {})
         return data
       end
 
-      if initialized?
+      if local_bucketing_initialized?
```

Comment:
do you need to rebase this? I thought this was changed in the previous PR?

Reply:
I changed it to `initialized` in the last pr and now I'm changing it again (lol) to `local_bucketing_initialized` cause I thought `initialized` was a little vague since the client handles both cloud and local. No, rebase needed